frontend/apps/reader/modules/readeruserhyph.lua

```diff
@@ -117,22 +118,22 @@ function ReaderUserHyph:updateDictionary(word, hyphenation)
         return
     end
 
-    local word_lower = self.ui.document:getLowercasedWord(word)
+    local word_lower = util.lowercaseString(word)
```

Comment:
But I'm fine if these become `cre.lowercaseString(word)`.
(But if there's a utf8proc ffi wrapper, even cleaner to use it instead.)